build.gradle.kts

```diff
@@ -10,10 +11,11 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 val modName = rootDir.name
 
 /**
- * Where your Starsector game is installed to.
+ * Where your Starsector game is installed to. Either set the system variable STARSECTOR_DIRECTORY to that path, or change
+ * the path behind the ?:.
  * Note: On Linux, if you installed Starsector into your home directory, you have to write /home/<user>/ instead of ~/
  */
-val starsectorDirectory = "C:/Program Files (x86)/Fractal Softworks/Starsector"
+val starsectorDirectory = System.getenv("STARSECTOR_DIRECTORY") ?: "C:/Program Files (x86)/Fractal Softworks/Starsector"
```

Comment:
```
if(providers.gradleProperty("starsector.dir").isPresent) providers.gradleProperty("starsector.dir").get()
```
might actually be better than System.getenv, since IntelliJ doesn't seem to care much about environment variables that were set on the system =/